src/sagemaker/serve/utils/hardware_detector.py

```diff
@@ -48,55 +48,56 @@
 }
 
 
-def _get_gpu_info(instance_type: str, session: Session) -> int:
+def _get_gpu_info(instance_type: str, session: Session) -> tuple:
```

Comment:
Type hint should be more explicit, ie. Tuple[int, int]

Reply:
Also, if we assume orders in Tuple, please make sure it's well documented.

Reply:
Great, will make it clear as below doc string
`
"""
    Get GPU info for the provided instance
    @return: Tuple containing: [0]number of GPUs available and [1]aggregate memory size in MiB
 """
`